exchanges/order/orders.go

```diff
@@ -1106,28 +1107,52 @@ func (c *Cancel) Validate(opt ...validate.Checker) error {
 	return nil
 }
 
-// Validate checks internal struct requirements
+// Validate checks internal struct requirements and returns filter requirement
+// options for wrapper standardization procedures.
 func (g *GetOrdersRequest) Validate(opt ...validate.Checker) error {
 	if g == nil {
 		return ErrGetOrdersRequestIsNil
 	}
+
 	if !g.AssetType.IsValid() {
 		return fmt.Errorf("%v %w", g.AssetType, asset.ErrNotSupported)
 	}
+
+	if g.Side == UnknownSide {
+		return errUnrecognisedOrderSide
+	}
+
+	if g.Type == UnknownType {
+		return errUnrecognisedOrderType
+	}
+
 	var errs common.Errors
 	for _, o := range opt {
 		err := o.Check()
 		if err != nil {
 			errs = append(errs, err)
 		}
 	}
-
 	if errs != nil {
 		return errs
 	}
 	return nil
 }
 
+// Filter reduces slice by optional fields
+func (g *GetOrdersRequest) Filter(exch string, orders []Detail) FilteredOrders {
+	filtered := make([]Detail, len(orders))
+	copy(filtered, orders)
+	FilterOrdersByPairs(&filtered, g.Pairs)
+	FilterOrdersByType(&filtered, g.Type)
```

Comment:
Having all of these requests are likely to have consequeces. See the implementation of this function with the following:

```go
if AnyType == orderType || len(*orders) == 0 {
		return
	}

	target := 0
	for i := range *orders {
		if (*orders)[i].Type == orderType {
			(*orders)[target] = (*orders)[i]
			target++
		}
	}
```
If you provide anything other than `ANY`, then the filter _must_ match, even if its `unknowntype` and therefore, we will filter any orders that do not populate that data in their API response. My next review will focus on the concequences. A simple fix would be to allow `FilterOrdersByType` and `FilterOrdersBySide` to always accept unpopulated fields, or for `GetOrdersRequest` to have an additional field which will allow non-populated fields to be returned